### DISCUSSION TITLE
Need to override standard put method implementation of pemission-management endpoint.

### DISCUSSION THREAD

**Discussion by QaziTanzeel** [abpframework/abp#8552] [Q&A] [answered]
Hi,

I am looking for implementation, that how I can override the default endpoints of abp. The intent is just to put some code before the execution of standard implementation and then base method which I have overridden. 
The endpoint I want to override is **permission-management** put method.

**Comment by ebicoglu** [comment, marked as answer]
see https://docs.abp.io/en/abp/latest/Customizing-Application-Modules-Overriding-Services#example-overriding-an-application-service

**Reply by QaziTanzeel** [reply]
Thank you will explore this overriding feature. 👍 💯
